pkgs/development/python-modules/cleanlab/default.nix

```diff
@@ -44,13 +44,38 @@ buildPythonPackage {
     pandas
   ];
 
+  # This is ONLY turned off when we have testing enabled.
+  # The reason we do this is because of duplicate packages in the enclosure
+  # when using the packages in nativeCheckInputs.
+  # Affected packages: grpcio protobuf tensorboard tensorboard-plugin-profile
```

Comment:
But why aren't they identical?

Reply:
I'm not sure, but it complained about duplicate packages in the enclosure without ``catchConflicts`` so I put the message there.

Reply:
This could be a symptom that we call the expression taking arguments from the wrong/inconsistent package sets?

Reply:
Ah, tensorflow is an input for that package.
Now it makes sense, it's because of it's custom protobuf, grpc, etc that it's likely propagating.

Reply:
Maybe we can make tensorflow use https://discourse.nixos.org/t/allowing-multiple-versions-of-python-package-in-pythonpath-nixpkgs/3849 `python-rewrite-imports` for its old protobuf - later